src/cdk/explorer/detectCdkProjects.ts

```diff
@@ -10,7 +10,7 @@ import { getLogger } from '../../shared/logger'
 import { CdkAppLocation } from './cdkProject'
 
 export async function detectCdkProjects(
-    workspaceFolders: ReadonlyArray<vscode.WorkspaceFolder> | undefined = vscode.workspace.workspaceFolders
+    workspaceFolders: readonly vscode.WorkspaceFolder[] | undefined = vscode.workspace.workspaceFolders
```

Comment:
?

Reply:
You're providing a default value. It'll never be undefined.

Reply:
This code is unrelated to `feature/debugconfig`

Reply:
This is a good sign that there was a change made in the feature branch that didn't belong there.

Reply:
Could have left this case, but then we have some files with `ReadonlyArray` and some with `readonly`.

We could avoid all of this trouble by releasing from a `stable` branch instead of `master`. Currently we don't have a common integration point.

Reply:
Disregard this: the reason is that `vscode.workspace.workspaceFolders` can be `undefined`: https://code.visualstudio.com/api/references/vscode-api#workspace